lib/index.js

```diff
@@ -539,7 +539,7 @@ function _getPublicKeyForPermissionCheck(publicKey, options, callback) {
   if(typeof publicKey === 'string') {
     publicKey = {id: publicKey || ''};
   }
-  api.getPublicKey(null, publicKey, function(err, publicKey) {
+  api.getPublicKey(publicKey, null, function(err, publicKey) {
```

Comment:
Oh, that sounds like another bug. Seems like it should be defaulting to `undefined`, not `null`.